### DISCUSSION TITLE
Setting viewport to use the default window size

### DISCUSSION THREAD

**Discussion by AutoQA23** [akshayp7/playwright-typescript-playwright-test#66] [Q&A]
Hi Akshay,
how to set browser.newPage({viewport: null})  in your framework? so the browsers are launched using default window size unless I have specified viewport for a specific project.
Thanks

**Comment by akshayp7** [comment]
Hi thanks for your patience can you check this for chrome in playwright.config.ts

{
      name: `Chrome`,
      use: {
        // Configure the browser to use.
        browserName: `chromium`,

        //Chrome Browser Config
        channel: `chrome`,
       contextOptions: {
             viewport: null, 
        },

        //Picks Base Url based on User input
        baseURL: testConfig[ENV],

        //Browser Mode
        headless: true,

        //Browser height and width
        viewport: { width: 1500, height: 730 },
        ignoreHTTPSErrors: true,

        //Enable File Downloads in Chrome
        acceptDownloads: true,

        //Artifacts
        screenshot: `only-on-failure`,
        video: `retain-on-failure`,
        trace: `retain-on-failure`,

        //Slows down execution by ms
        launchOptions: {
          slowMo: 0
        }
      },
    },
